Doc/library/asyncio-policy.rst

```diff
@@ -7,22 +7,29 @@
 Policies
 ========
 
-An event loop policy is a global per-process object that controls
-the management of the event loop. Each event loop has a default
-policy, which can be changed and customized using the policy API.
-
-A policy defines the notion of *context* and manages a
-separate event loop per context. The default policy
-defines *context* to be the current thread.
-
-By using a custom event loop policy, the behavior of
-:func:`get_event_loop`, :func:`set_event_loop`, and
-:func:`new_event_loop` functions can be customized.
+An event loop policy is a global (per-interpreter) object
+used to get and set the current :ref:`event loop <asyncio-event-loop>`,
+as well as create new event loops.
+The default policy can be :ref:`replaced <asyncio-policy-get-set>` with
+:ref:`built-in alternatives <asyncio-policy-builtin>`
+to use different event loop implementations,
+or substituted by a :ref:`custom policy <asyncio-custom-policies>`
+that can override these behaviors.
+
+The :ref:`policy object <asyncio-policy-objects>`
+gets and sets a separate event loop per *context*.
```

Comment:
Should/could *context* link to anything?
It's only mentioned here and in a couple of methods below, but it's otherwise not defined in this page.

Reply:
I had the same question and asked Guido that in #96377 and it seems there isn't really anything more to link it to; see [his reply](https://github.com/python/cpython/issues/96377#issuecomment-1258819222) for more details.

Reply:
Perhaps we should add -- in a separate PR -- a concise list of `asyncio`-related terms/concepts (coroutines, tasks, futures, event loops, transports, contexts, policies, ...).

Reply:
If we did so as a proper Sphinx `glossary`, they could be easily referenced with, e.g. `` :term:`coroutine` `` from anywhere (including via intersphinx).

Reply:
Good luck defining _context_. It has at least four unrelated uses/meanings:
- _context manager_, e.g. `TaskGroup` and `timeout`; see also `contextlib`
- a collection of _context variables_ (see `contextvars`) used to pass contextual state along to child tasks (e.g. `create_task(..., context=...)`)
- a dict with assorted things passed to [exception handlers](https://docs.python.org/3/library/asyncio-eventloop.html?highlight=asyncio%20exception%20handler#error-handling-api)
- The _context_ referred to by the docs for event loop policies